mongodb/driver/query/construct.xml

```diff
@@ -42,7 +42,7 @@
          <row>
           <entry>limit</entry>
           <entry><type>integer</type></entry>
-          <entry>Client side maximum number of documents to return</entry>
+          <entry>Maximum number of documents to return to the client</entry>
```

Comment:
Its not a number the server returns to the client (driver).
Its a number the driver returns to the caller.
In a server<->client relationship, the server can return plenty more. It is the client that closes its eyes and says "ney ney ney, there are no more results trallala lala la"